src/pages/Dashboard/container.js

```diff
@@ -213,6 +213,30 @@ const DashboardContainer = Wrapped =>
 
       return true;
     };
+    
+    checkIfManditoryEvent = (startDate) => {
+
+      // set the manditory dates (year is never checked)
+      const mandatoryDates = ['2019-00-01', '2019-05-27', '2019-12-25'];
```

Comment:
grand for now - but should these dates really be hardcoded?

Also the 27th May will not be the May bank holiday every year...

Reply:
This is to be manually updated on a yearly basis outside of the front end.

Reply:
I think these can be removed now. This data is now coming from the back-end. I can't remember if the front-end was updated to reflect this.

Reply:
This can be removed now. These dates come from the back-end now. I'm not sure if the front-end was updated to move away from this hard-coded stuff but if not it needs updated to use the back-end.